### PR TITLE
Mobile device

### PR DESCRIPTION
**Pull request by resdenia**
Add an Option to manage devices for user flow
reference is https://playwright.dev/docs/api/class-playwright#playwright-devices

### CODE REVIEW

File: client/src/containers/EditCodeContainer/index.tsx [hunk 1]
```diff
@@ -157,10 +174,23 @@ const EditCodeContainer: FunctionComponent<IProps> = ({
...
                         codeSnippet={codeSnippet}
                     />
-                    <EnvVariableWrapper
-                        uniqueKey={uniqueKey}
-                        onSetListEnvVariable={setEnvVariable}
-                    />
+                    <SideWrapper>
+                        <Text tag='h2'>Select Device</Text>
+                        <Text tag={'p'}>
+                            Choose devicewhere you want to run test
```
Comment (Doron-Bargo) [inline client/src/containers/EditCodeContainer/index.tsx]: Select the device on which you would like to execute the test.

File: client/src/containers/EnvVariableContainer/index.tsx [hunk 2]
```diff
@@ -101,10 +101,10 @@ const EnvVariableContainer: FunctionComponent<IProps> = ({
     };
     return (
         <EnvVariableWrapper>
-            <Text tag='h2'>Enviroment Variable</Text>
+            <Text tag='h2'>Secrets Management</Text>
             <Text tag={'p'}>
-                Add key-value pairs for the environment variables that you use
-                in the script.
+                Add key-value pairs for your secrets, that you can use in the
```
Comment (Doron-Bargo) [inline client/src/containers/EnvVariableContainer/index.tsx]: Include key-value pairs for your confidential information, which you can then employ as environment variables in your script.

File: helper/index-config.js [hunk 3]
```diff
@@ -1,4 +1,45 @@
...
+		let context = null;
+		let err = null;
+		let page = null;
+		let browser = null;
+
+		try {
+			browser = await playwright.launchChromium(false);
+			context = await browser.newContext({
+				recordHar: {
+					path: path.join(__dirname, '..', '..', 'tmp', 'example.har'),
```
Comment (Doron-Bargo) [inline helper/index-config.js]: Why we are working with a file called example?

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]
Great work please check my comments

File: helper/index-config.js (hunk 3 above)
Comment (resdenia) [reply helper/index-config.js]: updated to the page.har

**Review by Doron-Bargo — APPROVED** [review]
Great work

**Pull request merged**